packages/usa-range/src/index.js

```diff
@@ -0,0 +1,53 @@
+const selectOrMatches = require("../../uswds-core/src/js/utils/select-or-matches");
+const behavior = require("../../uswds-core/src/js/utils/behavior");
+
+const { prefix: PREFIX } = require("../../uswds-core/src/js/config");
+
+const RANGE_CLASSNAME = `${PREFIX}-range`;
+const RANGE = `.${RANGE_CLASSNAME}`;
+
+/**
+ * Update range callout for screen readers using the optional data attributes.
+ *
+ * Get optional data attributes, construct and appends aria-valuetext attribute.
+ *
+ * @example
+ *
+ * <input id="usa-range" class="usa-range" type="range" min="0" max="100" step="10" value="20" role="slider" data-text-unit="degrees">
+ *
+ * Callout returns "20 degrees of 100."
+ *
+ * @param {HTMLInputElement} targetRange - The range slider input element
+ */
+const updateCallout = (targetRange) => {
+  const rangeSlider = targetRange;
+  const defaultPrep = "of";
+  const optionalPrep = rangeSlider.dataset.textPreposition;
+  const prep = optionalPrep || defaultPrep;
+  const unit = rangeSlider.dataset.textUnit;
+  const val = rangeSlider.value;
+  const max = rangeSlider.getAttribute("max");
+
+  const callout = `${val} ${unit || ""} ${prep} ${max}`;
```

Comment:
This creates an extra empty space when there is no unit. For example, `20  de 100`. This doesn't appear to affect the readout in VoiceOver. I don't think this needs to be a blocker but wanted to flag it.

Reply:
Ooh that's a little annoying. Do you think we should update the unit tests to match? Or do you think I should revert to an `if/else` ?

Reply:
My preference is to edit the script so it outputs the expected number of spaces, but I don't want it to be a blocker. So if you can make it work, let's do it!

Reply:
I agree, I'd rather not have the extra space. My only other thought was adding a space to the `unit` value but that seemed messy... 

Resolved in [30e1d2b](https://github.com/uswds/uswds/pull/5472/commits/30e1d2b65d0389b012c4d00c52a46e985fe12592)